src/Ch9/Ch9.Shared/ViewModels/ShowViewModel.cs

```diff
@@ -69,7 +69,23 @@ public EpisodeViewModel[] Episodes
         public EpisodeViewModel SelectedEpisode
         {
             get => _selectedEpisode;
-            set => Set(() => SelectedEpisode, ref _selectedEpisode, value);
+            set
+            {
+	            Set(() => SelectedEpisode, ref _selectedEpisode, value);
+#if __IOS__
```

Comment:
I would extract this. Also, do you want to check for value instead of SelectedEpisode?

Why leave a TODO?

Reply:
Yep I'll amend this to do an extract method. Left a TODO as for unknown reason this code does not work (but should).